.yamato/ruamel/jobs/shared/utr_utils.py

```diff
@@ -37,3 +37,58 @@ def extract_flags(utr_flags, platform_name, api_name, build_config, color_space,
 
 
 
+
+def utr_standalone_not_split_flags(platform_spec, suite='playmode', platform='Standalone', testproject='.', editor_location='.Editor', artifacts_path=PATH_TEST_RESULTS, player_load_path='../../players', player_conn_ip='auto', scripting_backend='il2cpp', api_level='NET_4_6', color_space='Linear'):
```

Comment:
I think these functions got deleted with the PR where flags go to metafile?

Reply:
i thought i wrote a comment but it disappeared - thanks! i left these in while i was testing the merge but forgot to remove